### DISCUSSION TITLE
How to replace an arrow if there are more then one arrow between two components?

### DISCUSSION THREAD

**Discussion by AvanovaMaria** [Eliav2/react-xarrows#180] [Q&A]
Hello! I have a problem to solve: there are many links between some tables of a diagram. If I give same start and end tables names all the arrows between them are put one over each other, so I can't see all the possible links. How can I make gaps between those arrows?

@Eliav2  Thanks a lot for this project) xarrows are great!)

**Comment by seasick** [comment]
I don't think that there is a built-in solution for this.
I had a similar problem, but solved it by offsetting each arrow by a few pixels. Each incoming arrow to the same component would get an increased offset. Not the prettiest nor smartest solution, but it works in my case,

```
endAnchor: {
  position: 'auto',
  offset: {
    x: nextOffset(), 
  },
}
```
